pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -29,7 +29,7 @@ def test_cov(self, float_frame, float_string_frame):
         frame = float_frame.copy()
         frame["A"][:5] = np.nan
         frame["B"][5:10] = np.nan
-        result = float_frame.cov(min_periods=len(float_frame) - 8)
```

Comment:
the expected on the next line is not used either before being reassigned. are we missing an assert somewhere here?

Reply:
Thanks for pointing that out - have modified the test to do what I'd guess it was trying to